onnx/defs/math/defs.cc

```diff
@@ -1443,4 +1443,32 @@ ONNX_OPERATOR_SET_SCHEMA(
           matmulShapeInference(ctx, 0, 1);
         }));
 
+static const char* Round_ver11_doc = R"DOC(
+Round takes one input Tensor and rounds the values, element-wise, meaning
+it finds the nearest integer for each value.
+In case of halfs, the rule is to round them to the nearest even integer
```

Comment:
Nit: specify that the shape and type of the output is the same as input.

Reply:
I will